chi2/chinotype.py

```diff
@@ -344,6 +353,11 @@ def prepChi(self):
                 where concept_cd like 'KUMC|DischargeDisposition:%'
                 '''.format(self.chipats, schema)
                 cols, rows = do_log_sql(db, sql)
+                sql = '''
+                create index {0}_idx
+                on {0} (pn)
+                '''.format(self.chipats)
+                cols, rows = do_log_sql(db, sql)
```

Comment:
If we're going to join on `chipats`, we might as well index it. Though not sure we still need to keep using this code. On the other hand, maybe removing it can wait.